app/src/main/kotlin/com/wire/android/ui/calling/ongoing/OngoingCallViewModel.kt

```diff
@@ -70,7 +77,13 @@ class OngoingCallViewModel @Inject constructor(
                 observeCurrentCall()
             }
         }
-        showDoubleTapToast()
+        viewModelScope.launch {
+            delay(DELAY_TO_SHOW_DOUBLE_TAP_TOAST)
+            shouldShowDoubleTapToast = globalDataStore.getShouldShowDoubleTapToast(currentUserId.toString())
+            if (shouldShowDoubleTapToast) {
+                startDoubleTapToastDisplayCountDown()
+            }
+        }
```

Comment:
Couldn't this still be done inside the function that was removed? `showDoubleTapToast`